### PR TITLE
Upgrade to Akka HTTP 10.2.0

### PR DESCRIPTION
**Pull request by jrudolph**
+ some modernization

To be updated to 10.2.0 when released and only then merged

### CODE REVIEW

**Comment by jrudolph** [comment]
Tested locally, will fail until 10.2.0 is released to maven central

File: src/main/g8/default.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name=My Akka HTTP Project
 description=This is a seed project which creates a basic build for an Akka HTTP application using Scala.
-scala_version=2.13.1
-akka_http_version=maven(com.typesafe.akka, akka-http-core_2.13, stable)
+scala_version=2.13.3
+akka_http_version=10.2.0
```
Comment (ennru) [inline src/main/g8/default.properties:4]: 10.2.0 will become the stable version at Maven Central.

Comment (jrudolph) [reply src/main/g8/default.properties:4]: Yep, that should be reverted then.

Comment (jrudolph) [reply src/main/g8/default.properties]: Done

**Pull request merged**